app/views/dealerships/show.html.erb

```diff
@@ -7,4 +7,4 @@
 <p>Created at: <%= @dealership.created_at %></p>
 <p>Updated at: <%= @dealership.updated_at %></p>
 <p><a href='/dealerships/<%= @dealership.id %>/employees'>All <%= @dealership.name %> Employees</a></p>
-<p><%= link_to "Update #{@dealership.name}", "/dealerships/#{@dealership.id}/edit", method: :get %></p>
+<p><%= button_to "Update #{@dealership.name}", "/dealerships/#{@dealership.id}/edit", method: :get %></p>
```

Comment:
thanks for updating this to buttons that's more consistent with how I was doing these